qa/tasks/ceph_manager.py

```diff
@@ -1515,8 +1515,7 @@ def __init__(self, controller, ctx=None, config=None, logger=None,
         self.controller = controller
         self.next_pool_id = 0
         self.cluster = cluster
-        self.cephadm = cephadm
-        self.rook = rook
```

Comment:
undo the movement of these lines

Reply:
I think @rishabh-d-dave just wants to move all the variables needed to be overridden together.

Reply:
Exactly.